frontend/src/components/stats/ActiveNearAppsUsersByDate.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from "react";
+import ReactEcharts from "echarts-for-react";
+import * as echarts from "echarts";
+import { useTranslation } from "react-i18next";
+
+import { AccountsByDate } from "../../libraries/explorer-wamp/stats";
+
+import { Props } from "./TransactionsByDate";
+
+const ActiveNearAppsUsersByDate = ({ chartStyle }: Props) => {
+  const { t } = useTranslation();
+  const [activeAccountsByWeek, setWeekAccounts] = useState(Array());
+  const [week, setWeek] = useState(Array());
+
+  useEffect(() => {
+    // TODO(sparkyrider): Put a proper implementation in near-analytics after User Activity NEP lands and get implemented on indexer-for-explorer side.
```

Comment:
Should we open an issue to keep in mind that we need to make it proper way?

Reply:
Good point, I will do it before transitioning this issue to "ready for review" stage (so far there is an issue with the data itself, so we are not ready to merge it anyway)